src/metabase/server/middleware/log.clj

```diff
@@ -166,14 +172,16 @@
   ;; [async] wait for the pipe to close the canceled/finished channel and log the API response
   (a/go
     (let [result (a/<! chan)]
-      (log-info (assoc info :async-status (if (nil? result) "canceled" "completed"))))))
+      (log/with-context {:core-async-response? true}
+        (log-info (assoc info :async-status (if (nil? result) "canceled" "completed")))))))
 
 (defn- log-streaming-response [{{streaming-response :body, :as _response} :response, :as info}]
   ;; [async] wait for the streaming response to be canceled/finished channel and log the API response
   (let [finished-chan (streaming-response/finished-chan streaming-response)]
     (a/go
       (let [result (a/<! finished-chan)]
-        (log-info (assoc info :async-status (name result)))))))
+        (log/with-context {:streaming-response? true}
```

Comment:
I can take or leave these, in practice I am not sure if logging these response types in and of themselves is useful.